### PR TITLE
finish menu mobile

### PR DESCRIPTION
**Pull request by HanaSabih**
finish menu mobile

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      //});
+
+      document.querySelectorAll(".rightNav").forEach((n) =>
+        n.addEventListener("click", () => {
+          x.classList.remove("responsive");
+        })
+      );
+    </script>
+  </body>
+</html>
```
Comment (leonardodiasb) [inline test]: - [ ] Kindly remove the `test` file from your repository and please do not commit test files to your GitHub repository. It makes your repository ugly and does not add anything important to the main app. If you want to create files for testing you can have them on your local machine and you could use the `.gitignore` to not commit them.

File: style.css [hunk 2]
```diff
@@ -21,25 +25,129 @@ a {
...
+    display: block;
+    height: 755px;
+    border-bottom: 4px solid rgba(59, 59, 61, 0.08);
+    border-radius: 48px;
+    -webkit-border-radius: 48px;
+    -moz-border-radius: 48px;
+    -ms-border-radius: 48px;
+    -o-border-radius: 48px;
+    margin-bottom: 57px;
+  }
```
Comment (leonardodiasb) [inline style.css:117]: - [ ] Your mobile menu is working fine, but it is possible for the user to scroll down when the menu is open making it disappear from the screen and a bad user experience. Kindly fix this issue, you could make the mobile menu fixed or disable the scrolling behavior when the menu is open.
 - Your design:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @HanaSabih, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of Github flow. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test (hunk 1 above)
Comment (Abenezer-Tilahun) [reply test]: Fixed.

File: style.css (hunk 2 above)
Comment (Abenezer-Tilahun) [reply style.css:117]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @HanaSabih  @Thoth1111 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for implement the mobile menu 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**